app/components/auk/file-uploader.hbs

```diff
@@ -38,7 +38,7 @@
           {{on "click" this.clickInput}}
           {{on "key-up" this.clickInputOnEnter}}
         >
-          {{t "upload-files" numFiles=(if @multiple "multiple" "one")}}
+          {{t "upload-files" numFiles=(if @multiple "multiple" "1")}} 
```

Comment:
this used to work (difference between "bestand opladen", "bestanden opladen") but was no longer working for some reason.